cozy/cost_model.py

```diff
@@ -39,6 +43,37 @@ def composite_order(*funcs):
             return o
     return Order.EQUAL
 
+def unprioritized_order(funcs):
+    """Determine the Order between two maintenance cost of two expressions or
+    AMBIGUOUS at the first ambiguous result
+
+    Each argument should be a function that takes no argument and returns an
+    Order.
+    """
+    def flip(order):
+        """ Flips the direction of the Order """
```

Comment:
This function can be part of the `Order` type:

```python
class Order(Enum):
    # ...
    def flip(self):
        """Returns the opposite order"""
        if self == Order.LT:
            return Order.GT
        if self == Order.GT:
            return Order.LT
        return self
```

You can then invoke it as e.g. `assert Order.LT.flip() == Order.GT`